config_defaults_inc.php

```diff
@@ -977,12 +977,12 @@
  * os, os_build, platform, version, fixed_in_version, target_version, view_state,
  * summary, sponsorship_total, due_date, description, steps_to_reproduce,
  * additional_info, attachment_count, bugnotes_count, selection, edit,
- * overdue
+ * overdue, votes
  *
  * @global array $g_view_issues_page_columns
  */
 $g_view_issues_page_columns = array (
-	'selection', 'edit', 'priority', 'id', 'sponsorship_total',
+	'selection', 'edit', 'priority', 'votes', 'id', 'sponsorship_total',
```

Comment:
I have no strong objection (after all, this is customizable), but from my point of view, "count" determinates priority of the bug from exception-driven development point. That is why it is grouped with priority.